game/src/main/scala/io/aigar/game/Player.scala

```diff
@@ -7,9 +7,14 @@ class Player(val id: Int, startPosition: Vector2) {
   var cells = List(new Cell(0, startPosition))
 
   def update(deltaSeconds: Float, grid: Grid, players: List[Player]) {
-    val opponents = players.filterNot(_ == this)
-    cells.foreach { _.update(deltaSeconds, grid) }
-    cells.foreach { _.eats(opponents)}
+    if ( cells.isEmpty ) {
```

Comment:
As we are verbally discussing right now, it will stay like this since I only added the condition and the spawning.